test/integration/integration_suite_test.go

```diff
@@ -1,16 +1,14 @@
 package integration_test
 
 import (
-	"testing"
-	"time"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"testing"
 
 	types "github.com/onsi/ginkgo/v2/types"
 )
 
 func TestTelemetry(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Integration Suite Tests", types.ReporterConfig{SlowSpecThreshold: 10 * time.Second})
+	RunSpecs(t, "Integration Suite Tests", types.ReporterConfig{})
```

Comment:
This field `slowSpecThreshold` got deprecated see https://github.com/onsi/ginkgo/blob/v2.22.2/types/config.go#L234